Source/Scene/ArcGisMapServerImageryProvider.js

```diff
@@ -297,6 +301,91 @@ define([
         return url;
     }
 
+    function buildPickURL(imageryProvider, x, y, level, longitude, latitude) {
```

Comment:
Move these private functions to the bottom, after all the class stuff.

Reply:
Thanks, Have also move the existing private function buildImageUrl()

7aca562e32f4a173d1a418e909b7580703245a93